grpc/examples/src/main/protobuf/main.proto

```diff
@@ -0,0 +1,27 @@
+
+
+syntax = "proto3";
```

Comment:
is there a way to have this somehow formatted? it looks ugly ;)

Reply:
it's easy to format the generated code and push it to the repo, but improving printer/renderer is going to be more time-consuming so could we create a separate ticket for it?

Reply:
Sure :)